zeebe/scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorThread.java

```diff
@@ -103,13 +99,14 @@ private void doWork() {
     currentTask = taskScheduler.getNextTask();
 
     if (currentTask != null) {
-      final var actorName = currentTask.actor.getName();
-      try (final var timer = actorMetrics.startExecutionTimer(actorName)) {
+      final var metrics = currentTask.actor.metrics;
+      try (final var timer = metrics != null ? metrics.startExecutionTimer() : null) {
```

Comment:
:wrench: I would prefer a noop implementation here when disabled instead of null checks. It doesn't even have to allocate, can return the same dumb noop instance every time.

Reply:
I see the thumbs up but no changes? :smile:

Reply:
ops I did the noop implementation, but didn't hook it up here !